src/libtrx/include/libtrx/game/items/col.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "./types.h"
+
+int16_t Item_GetHeight(const ITEM *item);
+int32_t Item_GetDistance(const ITEM *item, const XYZ_32 *target);
+void Item_Translate(ITEM *item, int32_t x, int32_t y, int32_t z);
+bool Item_Test3DRange(int32_t x, int32_t y, int32_t z, int32_t range);
+
+bool Item_TestBoundsCollide(
+    const ITEM *src_item, const ITEM *dst_item, int32_t radius);
+const BOUNDS_16 *Item_GetBoundsAccurate(const ITEM *item);
+
+void Item_TakeDamage(ITEM *item, int16_t damage, bool hit_status);
+
+// Mesh_bits: which meshes to affect.
+// Damage:
+// * Positive values - deal damage, enable body part explosions.
+// * Negative values - deal damage, disable body part explosions.
+// * Zero - don't deal any damage, disable body part explosions.
+int32_t Item_Explode(int16_t item_num, int32_t mesh_bits, int16_t damage);
```

Comment:
Imo Item_Explode and Item_TakeDamage can be moved to a new module utils, the rest I think can stay. We'll be referring to the umbrella `items.h` include throughout the codebase anyway.

Reply:
Thanks, done :)